doc/mgr/prometheus.rst

```diff
@@ -145,7 +148,8 @@ Example configuration
 ---------------------
 
 This example shows a single node configuration running ceph-mgr and
-node_exporter on a server called ``senta04``.
+node_exporter on a server called ``senta04``. Note that this requires to add the
+appropriate instance label to every ``node_exporter`` target individually.
```

Comment:
We could add the relabel config in here for nodes to generate the shortname instance label by chopping off the end of the default instance label, and remove the explicit per-endpoint entry -- that part is orthogonal to the rest and doesn't seem to have any downsides to me

Reply:
Sounds good to me. I'm happy to propose somehting, maybe in a separate PR though?